node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -101,8 +101,10 @@ define(['jquery', 'oae.core'], function ($, oae) {
         var renderEditProfileMainPanel = function() {
             // If the display name is not valid, clear it to inform the template that the user
             // has no real display name
+            var tenantVisibility = oae.api.config.getValue('oae-principals', 'user', 'visibility');
             if (!isValidDisplayName()) {
                 profile.displayName = null;
+                profile.visibility = tenantVisibility;
```

Comment:
Can you just add a line of docs (and a reference to this ticket) to this ?

Note: the format for these docs would be:

```
// Comment
// @see https://github.com/oaeproject/3akai-ux/pull/4100
```